scripts/node/testAll.js

```diff
@@ -65,22 +68,29 @@ class TestAllScript extends Script {
   };
 
   runTest = packageName => {
-    if (!this.options.silentMode) {
+    if (!this.args.silent) {
       this.log(`\n${packageName}`);
     }
-    const result = this.exec(`yarn workspace @tupaia/${packageName} test`, {
-      printOutput: !this.options.silentMode,
-    });
-    if (this.options.silentMode) {
+
+    let passesTests = true;
+    try {
+      this.exec(`yarn workspace @tupaia/${packageName} test`, {
+        printOutput: !this.args.silent,
+      });
+    } catch (error) {
+      passesTests = false;
+    }
```

Comment:
This change to a try/catch seems unrelated? Was it broken before?

Reply:
Good point, I should have left a comment! This is because I switched from `child_process.spawnSync` to `child_process.execSync`, which throws an error instead of returning an error code and the `stderr` output. I've found that this behaviour is more JS-friendly, from a programming perspective

Reply:
Oh right! Makes sense, thanks for the explanation